### PR TITLE
feat: add C# samples

### PR DESCRIPTION
**Pull request by jsimonweb**

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -197,6 +236,26 @@ public static void listInfoTypes() throws IOException {
   }
 }
 {{< /tab >}}
+{{< tab header="C#" >}}
+public static ListInfoTypesResponse ListInfoTypes(string languageCode, string filter)
+{
+    var dlp = DlpServiceClient.Create();
```
Comment (kurtisvg) [inline content/en/_index.md:242]: Should we be showing how to use the `using` statement here as described delow?

Should it have the comment clarifying usage as well?

**Review by kurtisvg — CHANGES_REQUESTED** [review]

File: content/en/_index.md (hunk 1 above)
Comment (jsimonweb) [reply content/en/_index.md]: Updated.

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
Some change requests. Happy to discuss more.
Thanks for putting these together.

File: content/en/_index.md [hunk 2]
```diff
@@ -79,6 +79,18 @@ def example_snippet():
       # Snippet Content ...
 
 # [END product_example]
+{{< /tab >}}
+{{< tab header="C#" >}}
+// [START product_example]
+using System;
+namespace Example
```
[the remark this answers is not in the document]
Comment (jsimonweb) [reply content/en/_index.md]: Updated.

File: content/en/_index.md [hunk 3]
```diff
@@ -96,6 +108,12 @@ the sample work:
 * See https://cloud.google.com/compute/docs/quickstart-client-libraries before running the code snippet.
 */
 {{< /tab >}}
+{{< tab header="C#" >}}
+// Moves a persistent disk from one zone to another.
+//
```
[the remark this answers is not in the document]
Comment (jsimonweb) [reply content/en/_index.md]: Updated.

File: content/en/_index.md [hunk 4]
```diff
@@ -249,6 +321,25 @@ requests or if they are thread-safe).
...
+{{< /tab >}}
+{{< tab header="C#" >}}
+// Initialize client that will be used to send requests. This client only needs to be created
+// once, and can be reused for multiple requests. After completing all of your requests, call
+// the "Dispose" method on the client to safely clean up any remaining background resources,
+// or employ a "using" statement to do this automatically.
+using (var client = Server.CreateClient())
+{
+    // make a request with the client
+}
```
[the remark this answers is not in the document]
Comment (jsimonweb) [reply content/en/_index.md]: Updated.

File: content/en/_index.md [hunk 5]
```diff
@@ -249,6 +321,25 @@ requests or if they are thread-safe).
...
+
+// Another option is to use a "using" declaration, which will dispose of the resource
+// when execution falls outside the enclosing statement.
+static void ExampleMethod()
+{
+    using var client = Server.CreateClient();
+    // make a request with the client
+}
```
[the remark this answers is not in the document]
Comment (jsimonweb) [reply content/en/_index.md:352]: Updated.

File: content/en/_index.md (hunk 1 above)
Comment (jsimonweb) [reply content/en/_index.md:252]: Rolled back update based on feedback from amanda-tarafa@

Comment (jsimonweb) [reply content/en/_index.md:252]: Clarifying comments added.

File: content/en/_index.md [hunk 6]
```diff
@@ -249,6 +326,31 @@ requests or if they are thread-safe).
...
+
+// Example "using" statement.
+using (var client = Server.CreateClient())
+{
+    // make a request with the client.
+}
+
+// Example "using" declaration, which will dispose of the resource
+// when execution falls outside the enclosing statement block.
+static void ExampleMethod()
```
Comment (amanda-tarafa) [inline content/en/_index.md:348]:
```suggestion
public void ExampleMethod()
```

**Review by amanda-tarafa — APPROVED** [review]
One tiny nit, but otherwise LGTM

thanks!

**Review by kurtisvg — APPROVED** [review]

**Pull request merged**